### PR TITLE
Allow building 6l on unix

### PR DESCRIPTION
**Pull request by rminnich**
This is the first in a series of scripts that will let us build the toolchain on linux.

Adapted from NxM, but better: it's just two files for 6l, and it's buildable right in the tree.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
doing this in parallel with watching daniel's podcast :-) It's a really good podcast!

**Review by gmacd — APPROVED** [review]

**Comment by gmacd** [comment]
Any particular reason the build script is called BUILD?

File: sys/src/cmd/6l/BUILD [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+mkdir -p ../../../../linux_amd64/bin
```
Comment (dancrossnyc) [inline sys/src/cmd/6l/BUILD:2]: I'd greatly prefer that this just creates a binary called `6l.linux_elf64` or something and then we can copy it wherever needed.

**Comment by dancrossnyc** [comment]
> Any particular reason the build script is called BUILD?

Presumably because it builds the program. :-) But calling it `mklinux` or something would be more in keeping with the existing style.

**Pull request merged**